frontend/src/metabase-lib/v1/urls.ts

```diff
@@ -5,7 +5,7 @@ import type { ParameterWithTarget } from "metabase-lib/v1/parameters/types";
 import { getParameterValuesBySlug } from "metabase-lib/v1/parameters/utils/parameter-values";
 import { remapParameterValuesToTemplateTags } from "metabase-lib/v1/parameters/utils/template-tags";
 import { isTransientId } from "metabase-lib/v1/queries/utils/card";
-import type { ParameterId, ParameterValueOrArray } from "metabase-types/api";
+import type { ParameterValuesMap } from "metabase-types/api";
```

Comment:
:x: It's possible to use a disabled unit of time via URL

https://github.com/metabase/metabase/assets/6830683/9c8a88fb-a2c7-4a02-b21b-8677b539532d